src/components/form/Form.tsx

```diff
@@ -301,7 +311,7 @@ function HandleNavigationFocusComponent() {
         await navigateTo(focusNode.id, focusNode.baseId, nodeNavOptions);
       }
     })();
-  }, [navigateTo, focusNode, navigate, searchStringRef, exitSubform, validate, onFormSubmitValidation]);
+  }, [navigateTo, focusNode, navigate, searchStringRef, exitSubform, validate, onFormSubmitValidation, isNavigating]);
```

Comment:
Was `isNavigating` intentionally only used in the dependency array?

Reply:
Hehe no, I must have forgotten this one. I'll remove it!